src/3D/CreatureBody.h

```diff
@@ -0,0 +1,53 @@
+/*****************************************************************************
+ * Copyright (c) 2018-2023 openblack developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/openblack/openblack
+ *
+ * openblack is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include <map>
+
+#include <entt/entity/registry.hpp>
+
```

Comment:
Unnecessary big includes in a header
```suggestion
#include <entt/fwd.hpp>
```